docusaurus/docs-internals/editor-controls/data-controls/aiText.md

```diff
@@ -12,13 +12,27 @@ Example of `aiText` control:
+| `position?`      | `number`                                 |    -    | The position of the control in toolbar                                                                                                                                                                                                                                                                                                                                                                                          |
+| `devices?`       | `"all"` \| `"desktop"` \| `"responsive"` | `"all"` | Define the devices where the control will be rendered. `"all"` renders the control on all devices. `"desktop"` renders the control only on desktop devices. `"responsive"` renders the control on both tablet and mobile devices.                                                                                                                                                                                               |
+| `roles?`         | `Array<Role>`                            |    -    | Render the control only if the current user's role matches one of the roles in the provided array. <br /> <br /> **`type Role = "admin" \| "viewer" \| "editor" \| "designer" \| "manager"`**                                                                                                                                                                                                                                   |
+| `selectedValue?` | `string`                                 |    -    | The value that will be used as context for generating new text. You can hardcode a default context that will not be overridden by the input context.                                                                                                                                                                                                                                                                            |
+| `default?`       | `Default`                                |    -    | The default control value. <br/> <br/> <b>`Default: { value: string; }`</b> <br/> <br/> `value` - the control's custom initial value <br/>                                                                                                                                                                                                                                                                                      |
+| `style?`         | `function`                               |    -    | This function generates CSS output based on the value from the control. The parameter is an object containing a `value` key, which holds the current value of the control. The function returns an object with a CSS selector key and CSS property values.  <pre>`style: ({value}) => {`<br/> `return {`<br/>  `"{{WRAPPER}} .brz-text": {`<br/>   `display: value === "on" ? "flex" : "none"`<br/>  `}`<br/> `}`<br/>`}`</pre> |
```

Comment:
for CSS example, to be more specific better set:
`display: value === "" ? "none" : "block"`